spec/controllers/redirections_controller_spec.rb

```diff
@@ -3,25 +3,23 @@
 RSpec.describe RedirectionsController do
   describe "GET :next" do
     it "redirects to the next site" do
-      gabe = create(:redirection)
-      edward = create(:redirection, next: gabe)
-      gabe.update(next: edward)
+      gabe = Redirection.find_by!(slug: "gabe")
```

Comment:
Wait, do the seeds get created before each spec run or as part of `rake db:test:prepare`? I'm not used to accessing AR objects created outside of the spec block.